### PR TITLE
Use relative linking

### PR DESCRIPTION
**Pull request by pitaj**
ref: https://community.nodebb.org/topic/11428/profile-picture-cropping-modal-issue/

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -28,7 +28,6 @@ pidfile
 /public/templates
 
 /public/sounds
-/public/uploads
```
Comment (julianlam) [inline .gitignore]: This folder is to be ignored, why was it removed here?

File: src/file.js [hunk 2]
```diff
@@ -12,35 +12,76 @@ var utils = require('./utils');
...
+
 file.saveFileToLocal = function (filename, folder, tempPath, callback) {
 	/*
 	 * remarkable doesn't allow spaces in hyperlinks, once that's fixed, remove this.
 	 */
-	filename = filename.split('.');
-	filename.forEach(function (name, idx) {
-		filename[idx] = utils.slugify(name);
-	});
-	filename = filename.join('.');
```
Comment (julianlam) [inline src/file.js]: Please test to see if this block is still required. We don't even use remarkable anymore...

Comment (julianlam) [reply src/file.js]: ... nope it does not. I'll commit a fix separately.

**Comment by julianlam** [comment]
Now it has conflicts :trollface:

File: src/file.js (hunk 2 above)
Comment (pitaj) [reply src/file.js]: It's still there, I just changed it to use `.map` instead

**Pull request merged**